### PR TITLE
Patch 11

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Added a new section "Use the new EAC to configure message delivery restrictions"

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/configure-message-delivery-restrictions.md [hunk 1]
```diff
@@ -50,9 +50,37 @@ For additional management tasks related to recipients, see the following topics:
...
+- **Block messages from**: Use this section to block people from sending messages to this user.
+
+   - **None**: This option specifies that the mailbox won't reject messages from any senders in the Exchange organization. This is the default option.
+
+   - **Selected senders**: This specifies that the user can choose from a list of senders. Click ![Add Icon](../../media/ITPro_EAC_AddIcon.gif) **Add sender** to display the list of all recipients in your Exchange organization. You can also search for a specific recipient by typing the recipient's name in the search box. Select the desired recipients, and then click **Confirm**.
+
+5. Click **Save** to save your changes. Click **Close** to exit from the **Manage mail flow settings** display pane.
+
+
+## Use the Classic EAC to configure message delivery restrictions
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/configure-message-delivery-restrictions.md:81]:
```suggestion
## Use the classic EAC to configure message delivery restrictions
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/configure-message-delivery-restrictions.md [hunk 2]
```diff
@@ -50,9 +50,37 @@ For additional management tasks related to recipients, see the following topics:
...
+
+1. In the Classic EAC, navigate to **Recipients** \> **Mailboxes**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/configure-message-delivery-restrictions.md:83]:
```suggestion
1. In the classic EAC, navigate to **Recipients** \> **Mailboxes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**